src/seedu/addressbook/ui/MainWindow.java

```diff
@@ -22,6 +24,7 @@
 
     private Logic logic;
     private Stoppable mainApp;
+    private int theme = 0;
```

Comment:
Do you think that using of enum would improve the code quality?

Reply:
Yes it would make the code more friendly, especially without the use of comments. Thank you.